routingv8/routes.go

```diff
@@ -0,0 +1,39 @@
+package routingv8
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/url"
+)
+
+// Routes returns routes.
+// The required parameters for this resource is a origin location and a destination location.
+// See https://developer.here.com/documentation/routing-api/dev_guide/topics/send-request.html
+// for details about other parameters.
+func (s *RoutingService) Routes(
+	ctx context.Context,
+	req *RoutesRequest,
+) (_ *RoutesResponse, err error) {
+	u, err := s.URL.Parse("routes")
+	if err != nil {
+		return nil, err
+	}
+
+	values := make(url.Values)
+	values.Add("transportMode", "truck")
+	values.Add("return", "summary")
+	values.Add("origin", fmt.Sprintf("%v,%v", req.Origin.Lat, req.Origin.Long))
+	values.Add("destination", fmt.Sprintf("%v,%v", req.Destination.Lat, req.Destination.Long))
+
+	emptyJSONBody := []byte("{}")
+	r, err := s.Client.NewRequest(ctx, u, http.MethodPost, values.Encode(), emptyJSONBody)
```

Comment:
According to [this documentation](https://developer.here.com/documentation/routing-api/dev_guide/topics/send-request.html#send-a-request) we should be able to do a GET request. Think that's possible?
Then we wouldn't need an empty body either.